### ISSUE TITLE
Comma wrongly appearing outside quotation mark.

### ISSUE THREAD

**Issue by mccaskey** [Juris-M/citeproc-js#232]
In `<citation>` CSL for Chicago Manual of Style 17th edition (note), change the layout delimiter from a semicolon to a comma, that is, change `<layout suffix="." delimiter="; ">` to `<layout suffix="." delimiter=". ">`,  Select some journal articles. Ensure locale is en-US.

While the delimiting semicolon rightly appeared outside the closing quotation marks, a comma should appear inside. But it doesn’t. We get this citation

> Galen, “My own books”, Fumerton, “Logic”, Grafton, “Ancient Works.”

instead of this one

> Galen, “My own books,” Fumerton, “Logic,” Grafton, “Ancient Works.”

The period got moved inside, so we know `punctuation-in-quote` is active,

**Comment by zepinglee** [comment]
The test fixture [punctuation_SuppressPrefixPeriodForDelimiterSemicolon.txt](https://github.com/citation-style-language/test-suite/blob/master/processor-tests/humans/punctuation_SuppressPrefixPeriodForDelimiterSemicolon.txt) reflects this behavior. It seems to be a deliberate design.

**Comment by mccaskey** [comment]
That test fixture has shows `<citation><layout suffix="." delimiter="; ">`. Change the delimiter to a comma: `<citation><layout suffix="." delimiter=", ">`.  

`Doe, “His Anonymous Life”; Roe, “Her Anonymous Life.”` should become 
`Doe, “His Anonymous Life,” Roe, “Her Anonymous Life.”` with the comma inside the closing quoteation mark.

**Comment by fbennett** [comment]
Yes. Somewhere in the code, there is JS object with parameters for various combinations of punctuation and quotes. There's not much left to chance on that front.
